frontend/src/components/layouts/Navigation.tsx

```diff
@@ -39,6 +39,30 @@ const Navigation: React.FC<INavigationProps> = (props: INavigationProps) => {
   const pipelineNavItems: SideNavigationProps.Item[] = [
     { type: 'link', text: t('nav.home'), href: '/' },
     { type: 'link', text: t('nav.projects'), href: '/projects' },
+    {
+      type: 'link',
+      text: t('nav.analyticsStudio'),
+      href: '/analytics',
+      info: (
+        <Popover
+          header={
+            t('analytics:information.introducingAnalyticsHeader') ??
+            'Introducing analytics'
+          }
+          content={
+            <div>
+              {t('analytics:information.introducingAnalyticsContent')}
+              <br />
+              <Link href="#" variant="primary">
+                Learn more.
```

Comment:
i18n? And the `Learn more` Link?

Reply:
analytics studio docs link will add later.